openassessment/templates/openassessmentblock/staff_area/oa_staff_grade_learners.html

```diff
@@ -0,0 +1,31 @@
+{% load i18n %}
+
+<!-- staff-info__student is necessary to get proper styling for the staff assessment form -->
```

Comment:
I've removed all styling on staff-info__student, and am now styling on wrapper--input which is more generic.